examples/with-typescript/nuxt.config.ts

```diff
@@ -0,0 +1,41 @@
+import { Configuration } from "@nuxt/types";
+import { resolve } from "path";
+
+const config: Configuration = {
+  server: {
+    host: "0.0.0.0"
+  },
+  rootDir: resolve(__dirname, "../.."),
+  srcDir: __dirname,
+  buildDir: resolve(__dirname, ".nuxt"),
+  dev: false,
+  render: {
+    resourceHints: false
+  },
+  serverMiddleware: ["../api/auth"],
+  auth: {
+    strategies: {
+      local: {
+        endpoints: {
+          login: { propertyName: "token.accessToken" }
+        }
+      }
+    }
+  },
+  buildModules: ["@nuxt/typescript-build"],
+  modules: [
+    "bootstrap-vue/nuxt", // https://bootstrap-vue.js.org/docs/#nuxt-js
+    "@nuxtjs/axios",
+    "@nuxtjs/auth",
+    "@nuxtjs/toast",
+    "@@"
```

Comment:
I don't remember what `@@` is for 🤷‍♂️

Reply:
It is rootDir.

Reply:
So no changes here right? I pulled this from the other example I believe